app-misc/qmodmaster/qmodmaster-0.5.2.recipe

```diff
@@ -0,0 +1,87 @@
+SUMMARY="A free Qt-based implementation of a ModBus master application"
+DESCRIPTION="A graphical user interface allows easy communication with ModBus \
+ RTU and TCP slaves. QModMaster also includes a bus monitor for examining all  \
+ traffic in the bus
+ QModMaster is based on libmodbus. for modbus  \
+ communication and on QsLog for logging.
+ Supports Haiku, Windows and Linux.
+
+-Features
+     ModBus RTU/TCP Master
+     Monitor Bus activity [Raw Data and PDU details]
+"
+COPYRIGHT="2012-2020 Elbar"
+HOMEPAGE="https://sourceforge.net/projects/qmodmaster/"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://download.sourceforge.net/qmodmaster/qModMaster-code-0.5.2-3/qModMaster-code-$portVersion-3.zip
+CHECKSUM_SHA256="1754581e3059f14c31c584aa7abe1f1a602d1814f293823272b1158bbc179b81"
+SOURCE_FILENAME="qModMaster-code-$portVersion-3.zip"
+SOURCE_DIR="qModMaster-code-$portVersion"
+ADDITIONAL_FILES="qmodmaster.rdef.in"
+
+ARCHITECTURES="?x86_gcc2 x86 x86_64"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+        SECONDARY_ARCHITECTURES="x86"
```

Comment:
Through the whole recipe you have been using spaces for indents, please use tab(s) where needed:
```diff
-         SECONDARY_ARCHITECTURES="x86"
+	SECONDARY_ARCHITECTURES="x86"
```

Reply:
ok.

Reply:
Still seeing spaces in front of the lines, please revert to tabs

Reply:
seem ok now.
i'm curious.. how did you find it..? notepad++ not show the difference

Reply:
Need to do another local pull with your changes, but I'm still seeing spaces here, you can check this by looking at the recipe in a browser and then select some text:
![github](https://user-images.githubusercontent.com/16057090/113472471-cd2ec680-9463-11eb-9dbd-3964e1948afd.png)